### PR TITLE
Update readme.md and the Tables module docs

### PR DESCRIPTION
**Pull request by AlexJameson**

### CODE REVIEW

**Review by DokaRus — DISMISSED** [review]

File: docs/modules/table.md [hunk 1]
```diff
@@ -42,16 +42,16 @@ insertRow(offset: number)
 **Examples**
 
 ```js
-quill.table.insertRow(1); // insert row below
-quill.table.insertRow(0); // insert row above
-quill.table.insertRow(-3); // insert a row three rows above the current position
-quill.table.insertRow(3); // insert a row three rows below the current position
+quill.table.insertRow(1); // inserts row below
+quill.table.insertRow(0); // inserts row above
+quill.table.insertRow(-3); // inserts a row above 3 rows from the current position
```
Comment (RomanTsukanov) [inline docs/modules/table.md:47]:
```suggestion
quill.table.insertRow(-3); // inserts a row 3 rows above the current position
```

File: docs/modules/table.md [hunk 2]
```diff
@@ -42,16 +42,16 @@ insertRow(offset: number)
...
+quill.table.insertRow(3); // inserts a row below 2 rows from the current position
```
Comment (RomanTsukanov) [inline docs/modules/table.md:48]: The commands states 3, but the text says 2. So which is it? I use 3 in my suggestion.

```suggestion
quill.table.insertRow(3); // inserts a row 3 rows below the current position
```

File: docs/modules/table.md [hunk 3]
```diff
@@ -84,15 +84,15 @@ quill.table.insertRowBelow();
...
+Adds a column to the left or right of the current text cursor position. The position of the column depends on the `offset` parameter value.
 
 **Methods**
 
 ```js
 quill.table.insertColumn(1); // insert row to the right
 quill.table.insertColumn(0); // insert row to the left
-quill.table.insertColumn(-3); // insert a column by 3 columns to the left
-quill.table.insertColumn(3); // insert a column by 3 columns to the right
+quill.table.insertColumn(-3); // insert a column after 3 columns to the left from the current position
```
Comment (RomanTsukanov) [inline docs/modules/table.md:94]:
```suggestion
quill.table.insertColumn(-3); // insert a column 3 columns before the current position
```

File: docs/modules/table.md [hunk 4]
```diff
@@ -84,15 +84,15 @@ quill.table.insertRowBelow();
...
+quill.table.insertColumn(3); // insert a column after 2 columns to the right from the current position
```
Comment (RomanTsukanov) [inline docs/modules/table.md:95]: The same question here.

```suggestion
quill.table.insertColumn(3); // insert a column 3 columns after the current position
```

File: docs/modules/table.md [hunk 5]
```diff
@@ -42,16 +42,16 @@ insertRow(offset: number)
 **Examples**
 
 ```js
-quill.table.insertRow(1); // insert row below
-quill.table.insertRow(0); // insert row above
-quill.table.insertRow(-3); // insert a row three rows above the current position
-quill.table.insertRow(3); // insert a row three rows below the current position
+quill.table.insertRow(1); // inserts row below
```
Comment (RomanTsukanov) [inline docs/modules/table.md:45]:
```suggestion
quill.table.insertRow(1); // inserts a row below
```

File: docs/modules/table.md [hunk 6]
```diff
@@ -42,16 +42,16 @@ insertRow(offset: number)
...
+quill.table.insertRow(0); // inserts row above
```
Comment (RomanTsukanov) [inline docs/modules/table.md:46]:
```suggestion
quill.table.insertRow(0); // inserts a row above
```

File: docs/modules/table.md [hunk 7]
```diff
@@ -84,15 +84,15 @@ quill.table.insertRowBelow();
...
+Adds a column to the left or right of the current text cursor position. The position of the column depends on the `offset` parameter value.
 
 **Methods**
 
 ```js
-quill.table.insertColumn(1); // insert row to the right
-quill.table.insertColumn(0); // insert row to the left
-quill.table.insertColumn(-3); // insert a column by 3 columns to the left
-quill.table.insertColumn(3); // insert a column by 3 columns to the right
+quill.table.insertColumn(1); // inserts row to the right
```
Comment (RomanTsukanov) [inline docs/modules/table.md:92]:
```suggestion
quill.table.insertColumn(1); // inserts a column to the right
```

File: docs/modules/table.md [hunk 8]
```diff
@@ -84,15 +84,15 @@ quill.table.insertRowBelow();
...
+quill.table.insertColumn(0); // inserts row to the left
```
Comment (RomanTsukanov) [inline docs/modules/table.md:93]:
```suggestion
quill.table.insertColumn(0); // inserts a column to the left
```

**Review by RomanTsukanov — DISMISSED** [review]

File: README.md [hunk 9]
```diff
@@ -2,9 +2,9 @@
   <a href="https://js.devexpress.com/" title="DevExtreme-Quill">DevExtreme Quill</a>
 </h1>
 
-Devextreme-Quill is a fork of [Quill](https://quilljs.com/), based on the 2.0 branch.
+The **DevExtreme Quill** is a fork of [Quill](https://quilljs.com/), based on the 2.0 branch.
 
-Comparing to the current stable version of Quill, Devextreme-Quill provides basic table support and improve working with lists.
+Comparing to the current stable version of Quill, the **DevExtreme Quill** supports basic table operations and brings enhancements to lists rendering.
```
Comment (dirk-pieterse) [inline README.md:7]: "Comparing"? Why do you need to use a gerund here?

Comment (dirk-pieterse) [reply README.md:7]:
```suggestion
**DevExtreme Quill** supports basic table operations and enhances lists rendering. 
```

File: README.md [hunk 10]
```diff
@@ -22,7 +22,8 @@ Comparing to the current stable version of Quill, Devextreme-Quill provides basi
 
 ## Modules
 
-[Modules](/docs/modules.md) allow Quill's behavior and functionality to be customized. Several officially supported modules are available to pick and choose from, some with additional configuration options and APIs. Refer to their respective documentation pages for more details:
+[Modules](/docs/modules.md) allow users to customize the **DevExtreme Quill**'s behavior and functionality. The officially supported modules are the following:
```
Comment (dirk-pieterse) [inline README.md:25]:
```suggestion
[Modules](/docs/modules.md) allow users to customize the **DevExtreme Quill**'s behavior and functionality. The following are official modules:
```

File: docs/modules/table.md [hunk 11]
```diff
@@ -14,7 +14,7 @@ var quill = new DevExpress.Quill('#editor', {
 
 #### insertTable
 
-Adds a table into a current selection range
+Adds a table into the current selection range.
```
Comment (dirk-pieterse) [inline docs/modules/table.md:17]:
```suggestion
Adds a table to the current selection range.
```

File: docs/modules/table.md [hunk 12]
```diff
@@ -31,7 +31,7 @@ quill.table.insertTable(2, 2);
 
 #### insertRow
 
-Adds a row from above or below the current position. The position of the added row depends on the `offset` parameter value.
+Adds a row above or below the current text cursor position. The position of the row depends on the `offset` parameter value.
```
Comment (dirk-pieterse) [inline docs/modules/table.md:34]:
```suggestion
Adds a row above or below the cursor's position. The position of the row depends on the `offset` parameter value.
```

File: docs/modules/table.md [hunk 13]
```diff
@@ -84,15 +84,15 @@ quill.table.insertRowBelow();
 
 #### insertColumn
 
-Adds a column to the left or right of the current position. The position of the added column depends on the `offset` parameter value.
+Adds a column to the left or right of the current text cursor position. The position of the column depends on the `offset` parameter value.
```
Comment (dirk-pieterse) [inline docs/modules/table.md:87]: Same here

File: docs/modules/table.md [hunk 14]
```diff
@@ -68,7 +68,7 @@ quill.table.insertRowAbove();
 
 #### insertRowBelow
 
-Adds a row from below the current position.
+Adds a row below the current text cursor position.
```
Comment (dirk-pieterse) [inline docs/modules/table.md:71]: Same here

File: docs/modules/table.md [hunk 15]
```diff
@@ -192,7 +192,7 @@ quill.table.deleteColumn();
 
 #### deleteTable
 
-Delete the table at the current selection range.
+Deletes the table at the current selection range.
```
Comment (dirk-pieterse) [inline docs/modules/table.md:195]: Why do you need to add 'current' everywhere? Do you need to highlight the difference between a previous selection?

**Review by dirk-pieterse — APPROVED** [review]

File: README.md (hunk 9 above)
Comment (AlexJameson) [reply README.md:7]: To be honest, I just don't know how to say the same thing without gerund. We need to clarify, why we have our own fork, so I want to compare our fork's functionality with the original Quill.

File: docs/modules/table.md (hunk 15 above)
Comment (AlexJameson) [reply docs/modules/table.md:195]: There's an important thing. Quill operates with lots of selection ranges simultaneously, and users should understand that in this case the table will be deleted at the range that is declared inside this function.

Comment (AlexJameson) [reply docs/modules/table.md:195]: @DokaRus What's your opinion? Is it important that we highlight that it is the current range or we can just leave 'at the selection range'?

Comment (DokaRus) [reply docs/modules/table.md:195]: 'at the selection range' LGTM

File: README.md (hunk 9 above)
Comment (dirk-pieterse) [reply README.md:7]: I suggested an alternative version

**Review by dirk-pieterse — APPROVED** [review]

**Review by DokaRus — APPROVED** [review]

**Review by dirk-pieterse — DISMISSED** [review]

File: README.md [hunk 16]
```diff
@@ -2,9 +2,9 @@
   <a href="https://js.devexpress.com/" title="DevExtreme-Quill">DevExtreme Quill</a>
 </h1>
 
-Devextreme-Quill is a fork of [Quill](https://quilljs.com/), based on the 2.0 branch.
+The **DevExtreme Quill** is a fork of [Quill](https://quilljs.com/), based on the 2.0 branch.
 
-Comparing to the current stable version of Quill, Devextreme-Quill provides basic table support and improve working with lists.
+**DevExtreme Quill** supports basic table operations and enhances lists rendering.
```
Comment (RomanTsukanov) [inline README.md:7]:
```suggestion
In comparison with the original library, **DevExtreme Quill** supports basic table operations and enhances lists rendering. 
```

**Review by RomanTsukanov — APPROVED** [review]

**Pull request merged**